src/database/accessors/loadBundle.ts

```diff
@@ -0,0 +1,23 @@
+import { db } from '../db';
+import { loadTableMetrics } from './loadTableMetrics';
+import type { TinyPgParams } from 'tinypg';
+import type {
+  Bundle,
+} from '../../types';
+
+export const loadBundle = async <T extends object>(
+  tinyPgQueryName: string,
+  tinyPgQueryParameters: TinyPgParams,
+  tableName: string,
+): Promise<Bundle<T>> => {
+  const { rows: entries } = await db.sql<T>(
```

Comment:
After reflection, I'd like to move this logic to its own `loadObjects` function, which I think is going to be more flexible for future for non-bundle developer use cases.  I'd also want that function to verify the type of the response, which means it (and this) would take a type guard for the objects expected in the return.

Reply:
+1 to moving type verification here, but I'm not sure I follow your intentions for `loadObjects`; can you say more about where you'd want to use that?

Reply:
> can you say more about where you'd want to use that?

Yes!  I have it implemented in a WIP (which I was really hoping to get pretty and ready for an update to this PR before hitting the road but it is going to have to wait)

Basically, there are instances where we want to get collections of objects but NOT a bundle; in particular when loading collections of objects internally for the purposes of decorating other objects.  For this case I'm writing a `loadObjects` (well, it will probably be `getObjects` based on our future conversation about load vs get) which can be called in those places without the unnecessary "count" overhead / bundle wrapper.

If this doesn't make sense just stay tuned for the commit that adds it -- I might merge the commits later but can at least start with it being separate for the sake of the review process.

Reply:
I ended up just merging it into one commit, since I think it is part of a single complete change.